UndertaleModLib/Models/UndertaleGameObject.cs

```diff
@@ -373,6 +373,7 @@ public enum EventType : uint
         KeyRelease = 10, // subtype is key ID, see EventSubtypeKey
         Gesture = 11, // TODO: mapping is a guess // TODO: subtypes
         Asynchronous = 12, // TODO: mapping is a guess // TODO: subtypes
+        PreCreate = 14
```

Comment:
wait, 14? what was 13 then? I don't really remember... at least add an Unknown13 value here

Reply:
I don't know what 13 was, but I can say with confidence that 14 is PreCreate. Will add unknown 13.